src/lib/Scenes/Home/Components/Sales/Relay/SalesRenderer.tsx

```diff
@@ -1,25 +1,20 @@
 import { defaultEnvironment } from "lib/relay/createEnvironment"
 import React from "react"
 import { graphql, QueryRenderer } from "react-relay"
-import { PAGE_SIZE } from "../Components/LotsByFollowedArtists"
 
 export function SalesRenderer({ render }) {
   return (
     <QueryRenderer
       environment={defaultEnvironment}
       query={graphql.experimental`
         query SalesRendererQuery {
-          sales: sales(live: true, is_auction: true) {
-            ...Sales_sales
-          }
           viewer {
             ...Sales_viewer
+            ...LotsByFollowedArtists_viewer
```

Comment:
This shouldn’t actually be here, because the query renderer does not render this component directly, it renders the `Sales` component.